### ISSUE TITLE
Snippet code doesn't work?

### ISSUE THREAD

**Issue by icougil** [a85/POSTMan-Chrome-Extension#1363]
Hi!
I'm trying to use your extension (which is very very nice) and it seems that switching from one Language to another in the generation code (after sending the request) doesn't work.
Btw, I'm working on a Mac with Chrome 44.0.2403.155 (64-bit) and the latest version of Postman (3.0.8)
Best,

**Comment by abhijitkane** [comment]
I'm not able to reproduce this. Can you enable the DevTools (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/) and check if there are any errors in the console when you switch languages?

**Comment by icougil** [comment]
Ey! Thank you for your quick response.
Yes it seems that there is one error in the console. See below 
![image](https://cloud.githubusercontent.com/assets/2370634/9388746/b4f50ee8-4767-11e5-9a32-7f11d52a876a.png)

**Comment by abhijitkane** [comment]
Can you send the whole screenshot (including the line numbers on the right)?
Is this happening for all requests? Can you try creating a blank GET request to any url and trying?

**Comment by icougil** [comment]
Here you have the screenshot:

![image](https://cloud.githubusercontent.com/assets/2370634/9406642/be1b7912-4802-11e5-88a4-bf1244b04a67.png)

It seem it appears only with certain requests, because I've tried to ask for a simple page and it works. By the way, I'm using global and environment variables, may be this can cause the problem?

Best,

**Comment by abhijitkane** [comment]
Can you expand the "errors: Array[1]" line? Also, what's the simplest request that you are able to reproduce this issue with? If you can export a collection which has this request, it'll help fix the issue faster.

**Comment by icougil** [comment]
here it is:
![image](https://cloud.githubusercontent.com/assets/2370634/9407932/1dc00ba2-480e-11e5-86d3-9c4157e7e8e7.png)
I think that the simple URL that has some problem is related also with global and environment variables. For example:
{{endPoint}}/.../{{version}}/access.json/xxx
I think that the problem raises with the interpretation of this URL that it seems it is not well formed, but it is after applying the local & global vars.

**Comment by abhijitkane** [comment]
I used this as the url: `{{endPoint}}/.../{{version}}/access.json/xxx`, and was shown this snippet:

```
var http = require("http");

var options = {
  "method": "GET",
  "hostname": "",
  "port": null,
  "path": "%7B%7BendPoint%7D%7D/.../%7B%7Bversion%7D%7D/access.json/xxx",
  "headers": {}
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();
```

for NodeJS. I'm seeing valid snippets in other languages too. Are you seeing the problem for any URL with variables?

**Comment by icougil** [comment]
I don't know what's happening, but when I'm running this command for example to another endpoint seeems that is working ok. 
For example, launching a request like the next one will be returned ok: 
http://echo.jsontest.com/{{version}}/{{endpoint}}
I actually don't know what's happening really... :-/

**Comment by icougil** [comment]
I think I've found the cause of the problem: we are using variables that containts the value of the protocol (http://) and this is causing the problem. I mean, we are using a URL like this "{{endPoint}}/blablablabla" and because the variable 'endPoint' doesn't include the protocol (it contains only an IP) I think that because of that the snippet generator doesn't work. I've checked that including in the value the protocol the snippet generator works perfectly.
Best,
